lib/unbroken.js

```diff
@@ -90,13 +95,22 @@ class Checker {
     }
   }
 
+  getRelativeFilePath(filePath) {
+    return filePath.substr(this.options.dir.length + 1);

```

Comment:
can we use something in the path module instead of straight up substring? this could possibly break if someone ran this in a directory with DFS or symlinks (I know, unlikely :))